docs/source/color_processing/color-processing-introduction/color-processing-introduction.rst

```diff
@@ -4,12 +4,12 @@ FTC Color Processing - Introduction
 Overview
 --------
 
-As of FTC SDK 10.1, the *FIRST* Tech Challenge Software Development Kit now
-includes some Color Processing features from OpenCV, a popular and powerful
+The *FIRST* Tech Challenge Software Development Kit will soon
+include some Color Processing features from OpenCV, a popular and powerful
 open-source library for vision processing.
 
 Introduced for *INTO THE DEEP*, these new features will help FTC teams identify
```

Comment:
Remove "will" and the sentence is future proofed and still conveys the meaning intended.